en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md

```diff
@@ -506,26 +252,20 @@ You can download the ZIP file and extract the contents to get the project code.
 
```

Comment:
Is this file updated. Can you updated the zip file.

Reply:
Added now.